packages/nouns-webapp/src/pages/Fork/DeployForkButton.tsx

```diff
@@ -0,0 +1,171 @@
+      case 'PendingSignature':
+        setIsWaiting(true);
+        setErrorMessage('');
+        break;
+      case 'Mining':
+        setIsLoading(true);
+        setIsWaiting(false);
+        props.setDataFetchPollInterval(50);
+        break;
+      case 'Success':
+        setIsLoading(false);
+        setIsTxSuccessful(true);
+        props.refetchData();
+        break;
+      case 'Fail':
+        setErrorMessage(
+          (
+            <>
+              {state?.errorMessage}
+              <br />
+              <Trans>Please try again.</Trans>
+            </>
+          ) || <Trans>Please try again.</Trans>,
+        );
+        setIsLoading(false);
+        setIsWaiting(false);
+        break;
+      case 'Exception':
+        setErrorMessage(
+          (
+            <>
+              {state?.errorMessage}
+              <br />
+              <Trans>Please try again.</Trans>
+            </>
+          ) || <Trans>Please try again.</Trans>,
+        );
+        setIsLoading(false);
+        setIsWaiting(false);
+        break;
+    }
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, []);
+
+  useEffect(() => {
+    handleExecuteForkStateChange(executeForkState);
+  }, [executeForkState, handleExecuteForkStateChange]);
+
+  const modalContent = (
+    <div className={classes.transactionModal}>
+      <h2 className={classes.transactionModalTitle}>
+        <Trans>Deploy Fork</Trans>
+      </h2>
+      <p>
+        <Trans>Deploying Nouns fork and beginning the forking period</Trans>
+      </p>
+      <p
+        className={clsx(
+          classes.transactionStatus,
+          isLoading && classes.transactionStatusLoading,
+          isTxSuccessful && classes.transactionStatusSuccess,
+          errorMessage && classes.transactionStatusError,
+        )}
+      >
+        {isWaiting && (
+          <>
+            <img
+              src="/loading-noggles.svg"
+              alt="loading"
+              className={classes.transactionModalSpinner}
+            />
+            Awaiting confirmation
+          </>
+        )}
+        {isLoading && (
+          <>
+            <img
+              src="/loading-noggles.svg"
+              alt="loading"
+              className={classes.transactionModalSpinner}
+            />
+            Deploying
+          </>
+        )}
+        {isTxSuccessful && (
+          <>
+            <Trans>Success! The fork has been deployed.</Trans>
+            <br />
+            {executeForkState.transaction && (
+              <a
+                href={`${buildEtherscanTxLink(executeForkState.transaction.hash)}`}
+                target="_blank"
+                rel="noreferrer"
+              >
+                <Trans>View on Etherscan</Trans>
+              </a>
+            )}
+          </>
+        )}
+        {errorMessage && errorMessage}
+      </p>
+    </div>
+  );
+
+  return (
+    <>
+      {!props.isForkPeriodActive && props.isThresholdMet && (
+        <button
```

Comment:
Button should be disabled when no account is connected

Reply:
yep yep. good catch. updated here https://github.com/nounsDAO/nouns-monorepo/pull/745/commits/95c7cf24c95f79f99dae0b524cc7e88e737eb18f